src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -884,4 +884,10 @@ inline std::istream& operator>>(std::istream& is, Affinity& affinity) {
  * environment variable is set (as affinity is configured explicitly)
  */
 static constexpr Property<Affinity> affinity{"AFFINITY"};
+
+/**
+ * @brief The devices that the inference task been executed.
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<std::string> execution_devices{"EXECUTION_DEVICES"};
```

Comment:
can we also support this for HETERO?

Reply:
Hi ilya, for hetero, I want to test more with situations like 1) on multiple GPUs target 2) more hetero specific cases featured with affinity, I would prefer to track in a sperate PR with jira https://jira.devtools.intel.com/browse/CVS-92661